tests/test_login.py

```diff
@@ -1,19 +1,34 @@
-import unittest
 
 
-from trytond.tests.test_tryton import ModuleTestCase
-from trytond.tests.test_tryton import suite as test_suite
+from trytond.tests.test_tryton import DB_NAME
+from werkzeug.test import Client
+from trytond.wsgi import app
+from werkzeug.wrappers import Response
+from werkzeug.wrappers.json import JSONMixin, _JSONModule
 
 
-class AuthenticationDummyTestCase(ModuleTestCase):
-    'Test Authentication Dummy module'
-    module = 'authentication_dummy'
+class JSONResponse(JSONMixin, Response):
+    json_module = _JSONModule
 
-def test_login_dummy(self):
-    self.assertTrue(False)
 
-def suite():
-    suite = test_suite()
-    suite.addTests(unittest.TestLoader().loadTestsFromTestCase(
-            AuthenticationDummyTestCase))
-    return suite
+class LoginTestCase(object):
+
+    def test_login_dummy(self):
+
+        # GIVEN
+        client = Client(app, JSONResponse)
+
+        # WHEN
+        user_id, user_login = self.user
+        url = '/{}/'.format(DB_NAME)
+        result = client.post(
+            url,
+            json={
+                'method': 'common.db.login',
+                'params': [user_login, 'chupala']
```

Comment:
Even if there is no explicit Code of Conduct in this public repository, please try to avoid potentially offensive words :pray: